src/tito/cli.py

```diff
@@ -332,6 +332,11 @@ def __init__(self):
                 help="Custom arguments specific to a particular builder."
                     " (key=value)")
 
+        self.parser.add_option("--quiet", dest="quiet", action="store_true",
+                help="Suppress output from the build process.")
+        self.parser.add_option("--verbose", dest="verbose", action="store_true",
+                help="Expose more output from the build process.")
```

Comment:
Yeah, wasn't really sure what would be best -- since `rpmbuild` exposed both I figured we could too, to have three levels and keep the default the same as today.